cg/meta/upload/scout/scout_config_builder.py

```diff
@@ -27,24 +40,101 @@ def __init__(self, hk_version_obj: Version, analysis_obj: Analysis, lims_api: Li
         self.load_config: ScoutLoadConfig = ScoutLoadConfig(
             delivery_report=self.get_file_from_hk({HK_DELIVERY_REPORT_TAG})
         )
+        self.status_db = status_db
```

Comment:
Because we need to use `get_case_from_internal_id` if we are fetching the genome reference from statusDB. It is not absolutely required everywhere yet.

Reply:
Removed